### PR TITLE
BE-889 - Commitactivity testfile

### PR DESCRIPTION
**Pull request by walstork**
https://pinpt-hq.atlassian.net/browse/BE-889

**Description:**

Added unit tests

### CODE REVIEW

**Comment by josecordaz** [comment]
Please also add cases for errors.

File: pkg/git/commitactivity_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	answer string
+}
+
+type getJSONStrReasonTestTable struct {
+	data   string
+	answer string
+}
+
+func TestGetCommitReasons(t *testing.T) {
+	auxMap := make(map[string][]fileReasons)
```
Comment (jhaynie) [inline pkg/git/commitactivity_test.go]: this looks great as a first PR.  one thing we use is the testify assert library which makes it easier to do assertions.  no need to change the PR (unless you want to)...

you can do:

```
assert := assert.New(t)
assert.Equal("foo", "bar")
```

as an example.

**Comment by josecordaz** [comment]
We didn't actually find a way to make it fail.

File: pkg/git/commitactivity_test.go (hunk 1 above)
Comment (walstork) [reply pkg/git/commitactivity_test.go]: Hi Jeff, I've noticed that you are using that library on other test files and I'll try to implement it on my next test files, meanwhile I'll investigate more about such library.

**Pull request merged**